app/views/achievements/_grid.html.haml

```diff
@@ -4,8 +4,8 @@
     .col-md-3.achievement
       .panel.panel-default{class: achievement_class(achievement, player)}
         .panel-heading=achievement.title
-        .panel-body
-          =t(achievement.side)
-          ="(#{pluralize(achievement.points, t(:point))})"
+        .panel-body{class: "description"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Avoid defining `class` in attributes hash for static class names